cmd/metal-api/internal/service/network-service.go

```diff
@@ -263,28 +282,15 @@ func (r *networkResource) createNetwork(request *restful.Request, response *rest
 		return
 	}
 
-	prefixes := metal.Prefixes{}
-	// all Prefixes must be valid
-	for i := range requestPayload.Prefixes {
-		p := requestPayload.Prefixes[i]
-		prefix, err := metal.NewPrefixFromCIDR(p)
-		if err != nil {
-			r.sendError(request, response, httperrors.BadRequest(fmt.Errorf("given prefix %v is not a valid ip with mask: %w", p, err)))
-			return
-		}
-
-		prefixes = append(prefixes, *prefix)
+	var childPrefixLength = metal.ChildPrefixLength{}
+	for af, length := range requestPayload.DefaultChildPrefixLength {
```

Comment:
Either this needs only to be done for private super or the validation must be extended to validate this field for non-private super networks, too.

Reply:
done